vertx-mongo-client/src/main/java/io/vertx/ext/mongo/InsertManyOptions.java

```diff
@@ -0,0 +1,102 @@
+package io.vertx.ext.mongo;
+
+import io.vertx.codegen.annotations.DataObject;
+import io.vertx.core.json.JsonObject;
+
+/**
+ * Options for configuring inserts.
+ *
+ * @author <a href="mailto:tomasz.groch@gmail.com">Tomasz Groch</a>
+ */
+@DataObject
+public class InsertManyOptions {
```

Comment:
If this only contains one field "ordered" do we really need a whole new class for this? Can't a boolean param just be specified in the insertManyXXX methods?